html_css/scripts/slider.js

```diff
@@ -0,0 +1,46 @@
+export class Slider {
+	constructor(selector, data) {
+		this.$slider = document.querySelector(selector)
```

Comment:
do not quite understand the "$", if it is a private variable, they usually start with "_", in the future the "$" sign at the end of the variable will denote observers and subscripts in rxjs.